binheap/binheap.py

```diff
@@ -0,0 +1,90 @@
+#! /usr/bin/env python
+# Queue Python file for Efrain, Mark and Henry
+
+
+class Binheap(object):
+    def __init__(self):
+        self.items = []
+
+    def push(self, value):
+        """Add value to bottom of binheap."""
+        self.items.append(value)
+        if len(self.items) > 1:
+            self.sort_bottom()
+
+    def pop(self):
+        """Remove top value from the binheap."""
+        top = self.items[0]
+        self.sort_top()
+        return top
+
+    def parent(self, position):
+        """Return parent of current position."""
+        if position == 0:
+            raise AttributeError("At top of tree.")
+        return (position-1)/2
+
+    def child_L(self, position):
+        """Return left child of current position."""
+        cl = 2*position + 1
+        if cl > len(self.items)-1:
+            return None
+        else:
+            return cl
+
+    def child_R(self, position):
+        """Return right child of current position."""
+        cr = 2*position + 2
+        if cr > len(self.items)-1:
+            return None
+        else:
+            return cr
+
+    def find_max_child(self, position):
+        """Return largest value between R and L child of current position."""
+        ChildR = self.child_R(position)
+        ChildL = self.child_L(position)
+        try:
+            if self.items[ChildL] >= self.items[ChildR]:
+                return ChildL
+            else:
+                return ChildR
+        except (AttributeError, IndexError, TypeError):
```

Comment:
I can account for a possible `TypeError` and `IndexError` here.  Where would the `AttributeError` come from?